changelog.d/4658.bugfix

```diff
@@ -0,0 +1 @@
+Settings: Allow account deactivation when the account was created using SSO.
```

Comment:
The file name is not good.
The original issue has number 4685, not 4658.

Reply:
Damn, you've got sharp eyes! Thanks 🙏